projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="range-wrap">
+  <div class="range-slider">
+    <div class="min-max-slider" #minMaxSlider>
+      <input id="min" class="min range-slider-range" name="min" type="range" min="0" max="1000" #minSlider appCypressData="min-range"/>
```

Comment:
Is this 1000 meant to be the resolution? Why is it hard coded?

Reply:
fixed